src/widgets/TariffCombo.php

```diff
@@ -32,6 +32,11 @@ class TariffCombo extends Combo
 
     public $_primaryFilter = 'tariff_ilike';
 
+    /**
+     * @var string tariff owner
+     */
+    public $seller = null;
```

Comment:
```suggestion
    public $seller;
```

`null` – и так значение по умолчанию